### DISCUSSION TITLE
必须使用指令才可以发送消息

### DISCUSSION THREAD

**Discussion by Chosen-1st** [DreamVoid/MiraiMC#198] [问答 / Q&A]
`# 游戏内配置
# 此部分配置只会影响 Minecraft 服务器内的聊天
general:
  # 是否允许bStats统计
  allow-bStats: true

  # 是否必须使用命令才能向QQ群发送聊天消息
  require-command-to-chat: false

  # 使用命令时是否同时向服务器内发送聊天消息
  command-also-broadcast-to-chat: true

  # 可用的世界，玩家只有在下面列出的世界中聊天才会被发送到QQ群
  available-worlds:
    - IridiumSkyblock
    - IridiumSkyblock_nether
    - IridiumSkyblock_the_end

  # 是否将上面的列表用作黑名单，即玩家在上面列出的世界聊天不会发送到QQ群
  available-worlds-use-as-blacklist: false

  # QQ群消息广播到游戏内聊天的格式（可以使用样式代码）
  # 变量：
  # %groupname% - 群名称
  # %groupid% - 群号
  # %nick% - 发送者群名片
  # %qq% - 发送者QQ号
  # %message% - 消息内容
  in-game-chat-format: "&6[服务器QQ群]&b%nick%&r: %message%"

  # 是否允许控制台使用聊天命令
  allow-console-chat: false

  # 如果作为控制台发送到QQ群的聊天消息，控制台要叫什么名字
  console-name: "控制台"

  # 游戏聊天前缀（聊天需要带有指定前缀才能发送到QQ群）
  requite-special-word-prefix:
    #启用这个功能
    enabled: false

    # 聊天前缀（可设置多个，群聊天只有带下面的前缀之一才会发送到服务器）
    prefix:
      - "#"

  # 当群名片不存在时是否尝试获取昵称
  use-nick-if-namecard-null: true

  # 启用 MiraiMC 内置的QQ绑定
  use-miraimc-bind: true

  # 已绑定玩家向服务器发送聊天消息的格式
  # 如果玩家没有绑定，消息格式仍按照上方设置的格式发送
  # 内置变量仍然可以使用
  bind-chat-format: "&6[QQ]&r %vault_prefix%%player_name%%vault_suffix%: %message%"

# 机器人配置
# 此部分配置只会影响到QQ群的聊天
bot:
  # 指定使用的QQ号，只有这个QQ号接收到的群消息才会被处理
  # 请至少设置一个
  bot-accounts:
    - ***

  # 指定接收指定群的群消息，只有这个群接收到的消息才会被处理
  # 请至少设置一个
  group-ids:
    - ***

  # 群聊天前缀（聊天需要带有指定前缀才能发送到服务器）
  requite-special-word-prefix:
    #启用这个功能
    enabled: false

    # 聊天前缀（可设置多个，群聊天只有带下面的前缀之一才会发送到服务器）
    prefix:
      - "#"

  # 服务器消息发送到QQ群的格式（可以使用PlaceholderAPI变量）
  # 自带变量：
  # %player% - 玩家显示昵称
  # %message% - 消息内容
  group-chat-format: "[空岛] %player%: %message%"

  # 是否发送玩家进服退服消息
  send-player-join-quit-message: true

  # 进服消息
  # 自带变量：
  # %player% - 玩家显示昵称
  player-join-message: "%player% 进入了空岛服务器"

  # 玩家进服消息发送间隔，单位：秒
  # 在此间隔内多次进入不会多次发送群消息
  player-join-message-interval: 0

  # 退服消息
  # 自带变量：
  # %player% - 玩家显示昵称
  player-quit-message: "%player% 离开了空岛服务器"

  # 玩家退服消息发送间隔，单位：秒
  # 在此间隔内多次退出不会多次发送群消息
  player-quit-message-interval: 0

# 黑名单
# 无论上方如何设置，下方的黑名单如有匹配则不会进行任何操作（优先级最高）
blacklist:
  # 屏蔽的QQ，插件不会转发此处列出的QQ发出的消息
  qq:
    - 
  
  # 屏蔽的玩家名，插件不会转发此处列出的玩家的所有消息
  player:
    - 
  
  # 屏蔽词，插件不会转发包含下方列出的屏蔽词的消息
  word:
    - `

**设置没有错 但是不可以直接在服务器内发送消息 需要使用/qchat才可以发送消息 前缀开了加前缀也没用**

**Comment by DreamVoid** [comment]
复制配置文件并没有什么用

**Reply by Chosen-1st** [reply]
> 复制配置文件并没有什么用

但是后台并未有任何报错
